src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
@lemire @wesm @edani did this exchange happen? could someone send a summary to dev@parquet.apache.org ? if not I can start a thread with @nongli as well